README.md

```diff
@@ -36,6 +36,7 @@ At Babylon, we firmly believe that **transparency** is a core value that should
 | João Pereira            | [@NSMyself](https://twitter.com/nsmyself)                     |
 | Martin Nygren           |                                                               |
 | Michael Brown           | [@mluisbrown](https://twitter.com/mluisbrown)                 |
+| Olivier Halligon        | [@AliSoftware](https://twitter.com/aligatr)                   |
```

Comment:
Please add @inamiy as well while we're at it. Maybe he can confirm his squad?

Reply:
@inamiy I just added you as well, could you confirm your details, and add yourself in the proper squad in the table below too?

Reply:
My role is not fixed yet, so maybe after that will be fine 😀